packages/grid/test/column-groups-lit.test.js

```diff
@@ -0,0 +1,4 @@
+import '../theme/lumo/vaadin-grid-styles.js';
+import '../src/vaadin-lit-grid.js';
+import '../vaadin-grid-column-group.js';
```

Comment:
This probably should use `lit` import too.

Reply:
Good catch, done.